oleaster-runner/src/main/java/com/mscharhag/oleaster/runner/StaticRunnerSupport.java

```diff
@@ -127,7 +166,7 @@ public static void it(String text) {
 	 */
 	public static void xit(String text, Invokable block) {
```

Comment:
Why did you decided to add the `Invokable` to this method and ignoring it here instead of ignoring it in the `StaticRunnerSupport`?